src/main/java/seedu/address/logic/commands/DeleteCommand.java

```diff
@@ -23,7 +23,8 @@ public class DeleteCommand extends Command {
             + "Parameters: INDEX (must be a positive integer)\n"
             + "Example: " + COMMAND_WORD + " 1";
 
-    public static final String MESSAGE_DELETE_PERSON_SUCCESS = "Deleted Person: %1$s";
+    public static final String MESSAGE_DELETE_PERSON_SUCCESS = "Deleted successfully!\n"
+            + "Deleted participant info: %1$s";
```

Comment:
Very minor nitpick, is "info" necessary here? I'm fine with it either ways.

Reply:
Ya.. seems a bit redundant! I just got rid of it!